stable/datadog/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: datadog
-version: 1.30.7
-appVersion: 6.10.1
```

Comment:
6.12.0 is not released yet, we'll change this later in another PR

Reply:
ok, I will remove it.